xml/System.Text.Unicode/UnicodeRange.xml

```diff
@@ -33,10 +39,21 @@
         <Parameter Name="length" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="firstCodePoint">To be added.</param>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="firstCodePoint">The first code point in the range.</param>
+        <param name="length">The number of code points in the range.</param>
+        <summary>Creates a new <see cref="System.Text.Unicode.UnicodeRange" /> that includes a specified number of characters starting at a specified Unicode code point.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException"> 
+           <paramref name="firstCodePoint" /> lies outside of the Basic Multilingual Plane. 
+
+-or-           
+
+<paramref name="length" /> is less than zero.
```

Comment:
>is less than zero. [](start = 27, length = 18)

or the length will get the range beyond 0xFFFF

Reply:
I am not seeing you have added the suggested part

---
In reply to: [140906341](https://github.com/dotnet/docs/pull/3223#discussion_r140906341) [](ancestors = 140906341)